src/fusor/translator.py

```diff
@@ -581,16 +581,25 @@ async def translate(
             else InternalTandemDuplication
         )
 
-        strand1 = arriba.strand1.split("/")[1]  # Determine strand that is transcribed
-        strand2 = arriba.strand2.split("/")[1]  # Determine strand that is transcribed
-        if strand1 == "+":
-            gene1_seg_start = arriba.direction1 == "upstream"
+        if variant_type is InternalTandemDuplication:
+            # Assume breakpoint describes segment end if the event is an ITD
+            gene1_seg_start = False
+            gene2_seg_start = False
         else:
-            gene1_seg_start = arriba.direction1 == "downstream"
-        if strand2 == "+":
-            gene2_seg_start = arriba.direction2 == "upstream"
-        else:
-            gene2_seg_start = arriba.direction2 == "downstream"
+            strand1 = arriba.strand1.split("/")[
+                1
+            ]  # Determine strand that is transcribed
+            strand2 = arriba.strand2.split("/")[
+                1
+            ]  # Determine strand that is transcribed
+            if strand1 == "+":
+                gene1_seg_start = arriba.direction1 == "upstream"
+            else:
+                gene1_seg_start = arriba.direction1 == "downstream"
+            if strand2 == "+":
+                gene2_seg_start = arriba.direction2 == "upstream"
+            else:
+                gene2_seg_start = arriba.direction2 == "downstream"
```

Comment:
Is this related to the tests?

Reply:
@korikuzma Will do. Am I good to merge in this PR?